### PR TITLE
Add ArviZ Code of Conduct

### PR DESCRIPTION
**Pull request by canyon289**
Initial draft of Arviz CoC.

### CODE REVIEW

File: Code_of_Conduct.MD [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# ArviZ Community Code of Conduct
+
+ArviZ is an affiliated project of NumFocus and adopts the NumFocus Code of Conduct directly.
```
Comment (ColCarroll) [inline Code_of_Conduct.MD]:
```suggestion
ArviZ adopts the NumFOCUS Code of Conduct directly.
```

File: Code_of_Conduct.MD [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+In other words ArviZ is a part of the NumFocus community and we expect our community to
```
Comment (ColCarroll) [inline Code_of_Conduct.MD]:
```suggestion
In other words, we expect our community to
```

File: Code_of_Conduct.MD [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+treat others with kindness and understanding, just as they would in any other NumFOCUS project.
+
+
+# THE SHORT VERSION  
+Be kind to others. Do not insult or put down others. Behave professionally. Remember that harassment and sexist, racist, or exclusionary jokes are not appropriate for NumFOCUS.
```
Comment (ColCarroll) [inline Code_of_Conduct.MD]:
```suggestion
Be kind to others. Do not insult or put down others. Behave professionally. Remember that harassment and sexist, racist, or exclusionary jokes are not appropriate.
```

File: Code_of_Conduct.MD [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+All communication should be appropriate for a professional audience including people of many different backgrounds. Sexual language and imagery is not appropriate.
+
+NumFOCUS is dedicated to providing a harassment-free community for everyone, regardless of gender, sexual orientation, gender identity, and expression, disability, physical appearance, body size, race, or religion. We do not tolerate harassment of community members in any form.
```
Comment (ColCarroll) [inline Code_of_Conduct.MD]:
```suggestion
We are dedicated to providing a harassment-free community for everyone, regardless of gender, sexual orientation, gender identity, and expression, disability, physical appearance, body size, race, or religion. We do not tolerate harassment of community members in any form.
```

File: Code_of_Conduct.MD [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+Thank you for helping make this a welcoming, friendly community for all.
+
+
+# How to Submit a Report
+If you feel that there has been a Code of Conduct violation an anonymous reporting form is available.
+**If you feel your safety is in jeopardy or the situation is an emergency, we urge you to contact local law enforcement before making a report to NumFOCUS. (In the U.S., dial 911.)**
+
+NumFOCUS is committed to promptly addressing any reported issues. If you have experienced or witnessed behavior that violates the NumFOCUS Code of Conduct, please complete the form below to make a report.
```
Comment (ColCarroll) [inline Code_of_Conduct.MD]:
```suggestion
We are committed to promptly addressing any reported issues. If you have experienced or witnessed behavior that violates this Code of Conduct, please complete the form below to make a report.
```

File: Code_of_Conduct.MD [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+**REPORTING FORM:** https://numfocus.typeform.com/to/ynjGdT
+
+Reports are sent to the NumFOCUS Code of Conduct Enforcement Team (see below).
+
+You can view the Privacy Policy and Terms of Service for TypeForm here.
+The NumFOCUS Privacy Policy is here: https://www.numfocus.org/privacy-policy
+
+
+# Full Code of Conduct
+The full text of the NumFocus/ArviZ Code of Conduct can be found on Numfocus's website
```
Comment (ColCarroll) [inline Code_of_Conduct.MD]:
```suggestion
The full text of the NumFOCUS/ArviZ Code of Conduct can be found on NumFOCUS's website  
```

File: Code_of_Conduct.MD [hunk 7]
```diff
@@ -0,0 +1,34 @@
+# ArviZ Community Code of Conduct
+
+ArviZ is an affiliated project of NumFocus and adopts the NumFocus Code of Conduct directly.
+In other words ArviZ is a part of the NumFocus community and we expect our community to
+treat others with kindness and understanding, just as they would in any other NumFOCUS project.
```
Comment (ColCarroll) [inline Code_of_Conduct.MD]:
```suggestion
treat others with kindness and understanding.
```

File: Code_of_Conduct.MD (hunk 1 above)
Comment (ColCarroll) [inline Code_of_Conduct.MD]:
```suggestion
ArviZ adopts the NumFocus Code of Conduct directly.
```

File: Code_of_Conduct.MD (hunk 2 above)
Comment (ColCarroll) [inline Code_of_Conduct.MD]:
```suggestion
In other words, we expect our community to
```

File: Code_of_Conduct.MD (hunk 7 above)
Comment (ColCarroll) [inline Code_of_Conduct.MD]:
```suggestion
treat others with kindness and understanding.
```

File: Code_of_Conduct.MD (hunk 3 above)
Comment (ColCarroll) [inline Code_of_Conduct.MD]:
```suggestion
Be kind to others. Do not insult or put down others. Behave professionally. Remember that harassment and sexist, racist, or exclusionary jokes are not appropriate.
```

File: Code_of_Conduct.MD (hunk 4 above)
Comment (ColCarroll) [inline Code_of_Conduct.MD]:
```suggestion
ArviZ is dedicated to providing a harassment-free community for everyone, regardless of gender, sexual orientation, gender identity, and expression, disability, physical appearance, body size, race, or religion. We do not tolerate harassment of community members in any form.
```

File: Code_of_Conduct.MD (hunk 5 above)
Comment (ColCarroll) [inline Code_of_Conduct.MD]:
```suggestion
We are committed to promptly addressing any reported issues. If you have experienced or witnessed behavior that violates this Code of Conduct, please complete the form below to make a report.
```

File: Code_of_Conduct.MD (hunk 6 above)
Comment (ColCarroll) [inline Code_of_Conduct.MD]:
```suggestion
The full text of the NumFOCUS/ArviZ Code of Conduct can be found on NumFOCUS's website  
```

**Comment by canyon289** [comment]
It's pretty easy to be good when you're copying and pasting from other people :)

The suggestions are great, I incorporated most of them. In a couple the Github UI is throwing me off by highlighting some of the text red. For example *or* and *do not*. Does this mean something?

image

**Comment by aloctavodia** [comment]
LGTM!

**Comment by ahartikainen** [comment]
LGTM

**Comment by ColCarroll** [comment]
LGTM

**Pull request merged**